Modules/SharePointDsc/DSCResources/MSFT_SPUserProfileServiceApp/readme.md

```diff
@@ -4,9 +4,20 @@ This resource will provision an instance of the user profile service to the
 farm. It creates the required databases using the parameters that are passed
 in to it (although these are only used during the initial provisioning).
 
-The specified InstallAccount or PSDSCRunAsCredential has to be the Farm Account.
-This is done to ensure that the databases are created with the correct schema
-owners and allow the user profile sync service to operate correctly.
+The specified InstallAccount or PSDSCRunAsCredential shouldn't be the Farm Account.
+The resource will throw an error when it is. However, the FarmAccount parameter
+should be the Farm Account. The resource will throw an error if it is not. This is
```

Comment:
Was this section restored from an older release? Since the FarmAccount param is being deprecated, shouldn't this be reworded to reflect that?

Reply:
Since I initially added the FarmAccount parameter and later on replaced this with retrieving the account from the Managed Accounts, I forgot to update the Readme file. Will correct.